### DISCUSSION TITLE
automatic package categorization

### DISCUSSION THREAD

**Discussion by tiagocpontesp** [CyberShadow/aconfmgr#160] [Ideas]
using appstream maybe, to different filenames

would also be cool to not save any explicitly installed ones that are already required by other packages, as an opt

**Comment by CyberShadow** [comment]
> using appstream maybe, to different filenames

What's appstream?

Some distributions categorize their packages into groups like desktop, system, utility, etc. Arch Linux doesn't do this - it does have groups, but a list of groups doesn't really let you make a good decision about which one single file to put it in.

> would also be cool to not save any explicitly installed ones that are already required by other packages, as an opt

This will be done if you flag them on your system as "installed as dependency" (`pacman -D --asdeps`). (This doesn't work for optional dependencies.)

**Reply by tiagocpontesp** [reply]
https://github.com/ximion/appstream-generator

know of any premade script to mark all the ones you could mark as a dep and not change your system?

**Reply by tiagocpontesp** [reply]
i think it's only for "apps", but still. btw thank you -- this is super useful.
